arcsi/api/data.py

```diff
@@ -22,7 +22,7 @@ def uploaded_episodes_in_last_x_days():
     episodes_count = Item.query.filter(
         given_date - timedelta(days=last_days) <= Item.play_date
         ).filter(Item.play_date <= datetime.today()
-        ).filter(Item.archived == True).count()
```

Comment:
@tuz666 elegant! Adding features by deleting code ;)

Reply:
:DD thanks for review!!